assets/js/blocks/product-collection/inspector-controls/hand-picked-products-control.tsx

```diff
@@ -17,6 +17,25 @@ import {
  */
 import { QueryControlProps } from '../types';
 
+/**
+ * Decodes HTML entities in a string.
+ * Example:
+ * decodeHTMLEntities( 'foo &amp; bar' ) // 'foo & bar'
+ * decodeHTMLEntities( 'Hoodie &#8211; Black' ) // 'Hoodie – Black'
+ *
+ * @param {string} str - The string containing HTML entities.
+ * @return {string} - The decoded string.
+ */
+function decodeHTMLEntities( str?: string ) {
```

Comment:
Hey @imanish003 I think we already have a function that does this, check [here](https://github.com/woocommerce/woocommerce-blocks/blob/4020db4c10d4a3d984e2207285976434bb0b2a89/assets/js/atomic/blocks/product-elements/add-to-cart/product-types/variable/variation-attributes/attribute-select-control.tsx#L78). I think we should probably used that.

Reply:
Good catch @albarin. Thanks 🙏  Let me quickly change the code.

Reply:
Made the changes in https://github.com/woocommerce/woocommerce-blocks/pull/11927/commits/70ea5f1a679dd94a685cef39631e406574130d44